code/modules/reagents/chemistry/holder.dm

```diff
@@ -211,36 +211,44 @@ var/const/INGEST = 2
 			continue
 		if(!M)
 			M = R.holder.my_atom
+
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 			//Check if this mob's species is set and can process this type of reagent
 			var/can_process = 0
```

Comment:
But why would it? IPC's don't process the vast bulk of reagents; you could just have an empty override for their default proc, then override it for the rare specific dual ones they can process (ie: ants, acid, what have you).